src/MICore/Debugger.cs

```diff
@@ -379,8 +379,11 @@ public void Init(ITransport transport, LaunchOptions options)
             FlushBreakStateData();
 
             _transport.Init(this, options, Logger);
+        }
```

Comment:
For vscode, from the Json I don't think there is a great way to do it, but removing it from the schema will cause a squiggly in the ui. We could also print an output message if it is set stating it is deprecated. that would work in vs and vscode.